client/dashboard/sites/difm-lite-in-progress/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { useSuspenseQuery } from '@tanstack/react-query';
+import { Button, __experimentalHStack as HStack } from '@wordpress/components';
+import { sprintf, __ } from '@wordpress/i18n';
+import { addDays, isPast } from 'date-fns';
+import { useAnalytics } from '../../app/analytics';
+import { useLocale } from '../../app/locale';
+import { siteBySlugQuery } from '../../app/queries/site';
+import { siteDifmWebsiteContentQuery } from '../../app/queries/site-do-it-for-me';
+import { siteDomainsQuery } from '../../app/queries/site-domains';
+import { sitePurchasesQuery } from '../../app/queries/site-purchases';
+import { PageHeader } from '../../components/page-header';
+import PageLayout from '../../components/page-layout';
+import RouterLinkButton from '../../components/router-link-button';
+import { WPCOM_DIFM_LITE } from '../../data/constants';
+import { formatDate } from '../../utils/datetime';
+import { hasGSuiteWithUs, hasTitanMailWithUs } from '../../utils/domain';
+import type { Site } from '../../data/types';
+
+function WebsiteContentSubmitted( { site }: { site: Site } ) {
+	const { recordTracksEvent } = useAnalytics();
+	const { data: primaryDomain } = useSuspenseQuery( {
+		...siteDomainsQuery( site.ID ),
+		select: ( data ) => data.find( ( domain ) => domain.primary_domain ),
+	} );
+	const hasEmailWithUs =
+		primaryDomain && ( hasGSuiteWithUs( primaryDomain ) || hasTitanMailWithUs( primaryDomain ) );
+
+	const recordEmailClick = () => {
+		recordTracksEvent(
+			hasEmailWithUs
+				? 'calypso_dashboard_difm_lite_in_progress_email_manage'
+				: 'calypso_dashboard_difm_lite_in_progress_email_cta',
+			{
+				domain: primaryDomain?.domain,
+			}
+		);
+	};
+
+	return (
+		<PageLayout
+			header={
+				<PageHeader
+					title={ __( 'Your content submission was successful!' ) }
+					description={ sprintf(
+						// translators: %d is the number of business days it will take to build the site. Always greater than 1.
+						__(
+							'We are currently building your site and will send you an email when it’s ready, within %d business days.'
+						),
+						4
+					) }
+				/>
+			}
+			size="small"
+		>
+			<HStack spacing={ 4 } justify="start">
+				<RouterLinkButton variant="primary" to={ `/v2/sites/${ site.slug }/domains` }>
```

Comment:
I believe we don't need to spell out `/v2/` in the router links (also in line 62 below)